### DISCUSSION TITLE
How to control the initial size of the model?

### DISCUSSION THREAD

**Discussion by Yeafun** [sceneview/sceneform-android#284] [Q&A] [answered]
Thank you very much for the open source, I tried all the samples with great results and even put a 4 meters tall tyrannosaurus rex in our school, but I didn't want it to be that big initially because it was too close causing me to have to retreat a long way to see it

**Comment by grassydragon** [comment, marked as answer]
Hi!
You can use the Filament bounding box for scaling the model: https://github.com/SceneView/sceneform-android/discussions/243#discussioncomment-1832858

**Comment by ThomasGorisse** [comment]
You can also move to https://github.com/SceneView in order to have a model auto scaling and auto centering.
